app/interactors/update_user.rb

```diff
@@ -0,0 +1,37 @@
+class UpdateUser
+  include Interactor
+
+  before :check_user_details
+  before :find_user
+  delegate :user, to: :context
+
+  def call
+    User.transaction do
+      update_user
+      user.save!
+    end
+    context.message = I18n.t('users.update.success')
+  end
+
+  protected
+
+  def find_user
+    context.user = User.find(context.id)
+  end
+
+  def update_user
+    user.update_attributes(user_params)
+  end
+
+  def user_params
+    context.to_h.slice(:email, :name, :is_active)
+  end
+
+  def check_user_details
```

Comment:
Since you removed this from the create interactor, do you think you should remove this as well?